app/src/main/java/org/mozilla/fenix/home/tabs/TabsComponent.kt

```diff
@@ -36,13 +38,18 @@ class TabsComponent(
     }
 }
 
-data class TabsState(val sessions: List<Session>) : ViewState
+data class TabsState(val sessions: List<SessionViewState>) : ViewState
+data class SessionViewState(val id: String, val url: String, val selected: Boolean)
+
+fun Session.toSessionViewState(selected: Session?): SessionViewState {
+    return SessionViewState(this.id, this.url, this.id == selected?.id)
```

Comment:
Instead of passing in selected as a `Session` here. Should we evaluate this outside this function and pass in `isSelected`.

Though I see why we're doing it here. I'm not a big fan of the `TabViewState` knowing whether or not it's selected as it becomes possible for two `TabViewState`'s to be selected.